src/helpers.js

```diff
@@ -0,0 +1,327 @@
+import chalk from 'chalk';
+import figlet from 'figlet';
+import { join, dirname } from 'path';
+import { existsSync } from 'fs';
+import { exec } from 'promisify-child-process';
+import ora from 'ora';
+
+//-------------------------------------------------------
+// Helpers
+//-------------------------------------------------------
+
+/**
+ * Success message.
+ *
+ * @param {string} msg Message to output.
+ */
+const error = (msg) => console.log(`${chalk.white.bgRed('Error')}${chalk.red(' - ')}${msg}`);
+
+/**
+ * Installs composer package.
+ *
+ * @param {string} projectPath Path to the project.
+ * @param {string} packageToInstall Name of the package to install.
+ */
+const installComposerPackage = async (projectPath, packageToInstall) => {
+	exec(`cd "${projectPath}" && composer require ${packageToInstall} --no-interaction`)
+};
+
+/**
+ * Check Node.js version
+ */
+const checkNodeVersion = async () => exec(`node -v`);
+/**
+ * Check Composer version
+ */
+const checkComposerVersion = async () => exec(`composer -v`);
+/**
+ * Check WP-CLI version
+ */
+const checkWPCliVersion = async () => exec(`wp --info`);
+
+/**
+ * Check Git version
+ */
+const checkGitVersion = async () => exec(`git --version`);
+
+//-------------------------------------------------------
+// Exports
+//-------------------------------------------------------
+
+/**
+ * Finds the root directory of the project by searching for wp-config.php.
+ *
+ * @param {string} currentDir Current directory.
+ */
+const findProjectRoot = (currentDir) => {
+	if (!currentDir || currentDir === '/') {
+		// Reached the root without finding wp-config.php
+		return null;
+	}
+
+	const wpConfigPath = join(currentDir, 'wp-config.php');
+	const wpConfigExists = existsSync(wpConfigPath);
+
+	if (wpConfigExists) {
+		// Found wp-config.php, return the current directory as the project root
+		return currentDir;
+	}
+
+	// Move up one directory level and continue the search
+	const parentDir = dirname(currentDir);
+	return findProjectRoot(parentDir);
+};
+
+/**
+ * Gets the output path for the project.
+ *
+ * @param {string} dirname Directory name.
+ */
+export const getProjectOutputPath = (dirname) => {
+	const rootDir = findProjectRoot(process.cwd());
+
+	if (!rootDir) {
+		error('Unable to find project root.');
+		process.exit(1);
+	}
+
+	return join(rootDir, 'wp-content', dirname);
+};
+
+/**
+ * Empty the console.
+ */
+export const clearConsole = () => {
+	process.stdout.write('\x1Bc');
```

Comment:
Can't this just be `console.clear()`?

Reply:
I have no idea this was taken from the previous version

Reply:
Ok, we'll deal with it later 😄